lib/node_modules/@stdlib/assert/has-same-constructor/test/test.js

```diff
@@ -0,0 +1,76 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var hasSameConstructor = require( './../lib' );
+
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	t.ok( true, __filename );
+	t.strictEqual( typeof hasSameConstructor, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'the function returns `true` if provided two arguments which have the same constructor', function test ( t ) {
+    var values;
```

Comment:
Please make sure you have installed and setup EditorConfig. You have mixed tabs and spaces throughout this file. You can find more instructions in our contributing and development guides.